cheroot/server.py

```diff
@@ -1375,7 +1379,7 @@ def resolve_peer_creds(self):  # LRU cached on per-instance basis
             raise NotImplementedError(
                 'UID/GID lookup can only be done under UNIX-like OS'
             )
-        elif not self.peercreds_resolve_enabled:
+        elif not self.peercreds_resolve_enabled or 'grp' not in globals() or 'pwd' not in globals():
```

Comment:
This violates the linters: https://travis-ci.org/cherrypy/cheroot/jobs/453609030

Reply:
Is this better?

```python
try:
    import grp
except ImportError:
    grp = None
```

And then later

```python
if grp is None:
    raise RuntimeError('...')
```
Do you have any reference for such a design pattern? There is no PEP about it.

Reply:
P.S. you can also add some boolean var with a proper name as a flag for whether grp/pwd are accessible, this tends to make code better readable.

Reply:
Too late. Please change it on your own preference, if you don't like the actual changes.